oceannavigator/frontend/src/components/WarningBar.jsx

```diff
@@ -24,7 +24,7 @@ export default class WarningBar extends React.PureComponent {
     if(this.props.showWarningInfo){
       return (
         <Alert bsStyle="warning" onDismiss={this.handleDismiss}>
-              The Ocean Navigator will go down for brief maintenance at 12:45 NST
+              {_("The Ocean Navigator will go down for brief maintenance at 12:45 NST")}
```

Comment:
Could we throw in a date too?

Reply:
Sure, but this section is supposed to be updated when your doing a rollout.
Seperate from this issue?

Reply:
Added the date, but we will have to update the current public branch for this to actually be usefule